cmp/ag-grid/AgGrid.scss

```diff
@@ -351,78 +351,80 @@
   }
 }
 
-// Ensure ag-grid popups (context menus, tooltips, and column filter controls) with default z-index of 5
-// appear over bp dialogs with z-index of 20.
-.ag-theme-balham.ag-popup,
-.ag-theme-balham-dark.ag-popup {
-  .ag-popup-child {
-    z-index: 30;
+body.xh-app {
```

Comment:
Should we also be wrapping the selector that begins on line 30 and makes up the bulk of our ag-grid overrides?

Reply:
Spoke to @haynesjm42 and he reports that doing so ends up making our `AgGrid.scss` overrides more specific than some corresponding `Grid.scss` overrides -- we didn't really want to go any further down the rabbit hole on that now.